vcx/wrappers/node/demo/faber.js

```diff
@@ -215,7 +225,7 @@ async function runFaber (options) {
   if (proof.proofState === ProofState.Verified) {
     logger.info('Proof is verified')
   } else {
-    logger.info('Could not verify proof')
+    logger.info('Proof verification failed, credential revoked')
```

Comment:
When does this execute? Seems like the revocation flow was supposed to exit on line `214`